pkg/riff/commands/namespace.go

```diff
@@ -153,3 +154,53 @@ func NamespaceCleanup(c *core.Client) *cobra.Command {
 
 	return command
 }
+
+func NamespaceAddCredentials(c *core.Client) *cobra.Command {
+	options := core.NamespaceAddCredentialsOptions{}
+
+	command := &cobra.Command{
+		Use:     "add-credentials",
```

Comment:
Consider renaming this to `set-credentials`, not all operations are adding a new secret, sometimes a user will want to update existing credentials. In HTTP semantics, this should be a PUT, not a POST.

Or better yet to avoid the hyphenation which we don't have anywhere else in the CLI `credentials set`

Reply:
A `riff credentials` command is what I was arguing for :)

Reply:
Not sure about the `set`. For now, this command adds a secret only if there is none matching.

Reply:
Updated. This command now allows secret updates. `riff namespace init` semantics remains unchanged.